lib/video_info/providers/aparat.rb

```diff
@@ -0,0 +1,137 @@
+class VideoInfo
+  module Providers
+    class Aparat < Provider
+      def self.usable?(url)
+        url =~ /((.*)aparat\.com\/v\/(.*))/
+      end
+
+      def provider
+        'Aparat'
+      end
+
+      def api_key
+        ""
+      end
+
+      %w[description].each do |method|
+        define_method(method) { _video[method] }
+      end
+
+      %w[duration width height].each do |method|
+        define_method(method) { _video[method].to_i }
+      end
+
+      def title
+        _video['title']
+      end
+
+      def author
+        _video['username']
+      end
+
+      def author_thumbnail_id
+        author_uri = _video['username']
+        @author_thumbnail_id ||= _parse_picture_id(author_uri)
+      end
+
+      def author_thumbnail(width = 75)
+        _video['profilePhoto']
+      end
+
+      def thumbnail_id
+        @thumbnail_id = "123"
+      end
+
+      def thumbnail(width = 200, height = nil)
+      return _video['big_poster']
+      end
+
+      def thumbnail_small
+        thumbnail(100, 75)
+      end
+
+      def thumbnail_medium
+        thumbnail(200, 150)
+      end
+
+      def thumbnail_large
+        thumbnail(640)
+      end
+
+      def keywords
+        keywords_array.join(', ')
+      end
+
+      def keywords_array
+        _video['tags'].map { |t| t['tag'] }
+      end
+
+      def embed_url
+        "http://www.aparat.com/video/video/embed/videohash/#{_video['uid']}/vt/frame/"
+      end
+
+      def date
+        Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.